### ISSUE TITLE
Bower is out of date

### ISSUE THREAD

**Issue by lewissmallmedia** [pavelk2/social-feed#137]
I just installed from bower and had a few problems then i re-install direct from git source and the issues disappeared. Is you bower the latest version. 

Cheers,

Lewis

**Comment by kamilon50** [comment]
Hello there how did you manage to install with bower ?

**Comment by lewissmallmedia** [comment]
yes but when looking at the code of the two jquery.socialfeed.js files (the one from github and the other from bower) they seem to differ. bower version has 564 lines and github version  is 747 lines

**Comment by lewissmallmedia** [comment]
Also looks like the latest code for time_ago is defaulting to Italian. 

I fixed on my local by simply putting moment.locale('en') at the top of the code.

Might be good to add a feature of changing default language in the settings.

**Comment by kamilon50** [comment]
​Good evening Pavel

Thank you for response to my previous email :-)

Is it ok to contact you tomorrow if I will have any more difficulties ?
​

All the Best,

Kamil
| CKdesign
http://t.sidekickopen28.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX43MPfv6Vd0yGC7fK95-VRJgRK56dPKmf6tNWk202?t=http%3A%2F%2Fckdesign.eu%2F&si=4632408293113856&pi=c12366d3-e0b3-45f1-c88b-44d3c9ec4186

**Comment by kamilon50** [comment]
Sorry not Pavel Lewis

All the Best,

Kamil
| CKdesign
http://t.sidekickopen28.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX43MPfv6Vd0yGC7fK95-VRJgRK56dPKmf6tNWk202?t=http%3A%2F%2Fckdesign.eu%2F&si=4632408293113856&pi=54b65703-3019-4a1d-ec04-c53aa40ee325

**Comment by Webarkitekt** [comment]
@lewissmallmedia to change the default moment lang you should load the moment locale you need.
Check out the documentation 

``` html
<!-- Moment Locale to format the date to your language (eg. italian lang)-->
<script src="bower_components/moment/locale/it.js"></script>
```

If you remove this line it will be set to default lang ('en')

Sorry but actually, the bower version is not up to date.

**Comment by pavelk2** [comment]
We need to update the version available through Bower to close this issue
